Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppProtectionPolicyiOS/MSFT_IntuneAppProtectionPolicyiOS.schema.mof

```diff
@@ -25,6 +25,13 @@ class MSFT_IntuneAppProtectionPolicyiOS : OMI_BaseResource
     [Write, Description("N/A")] Boolean PrintBlocked;
     [Write, Description("N/A")] Boolean FingerprintBlocked;
     [Write, Description("N/A")] Boolean FaceIdBlocked;
+    [Write, Description("N/A")] Boolean IsAssigned;
```

Comment:
I am aware that this is the same for the already existing parameters, but do you have a description for these new parameter (and if possible also the already existing parameters 😉)? That way the generated documentation will be more useful.

Reply:
@Jamiet19 can you look at providing a description as highlighted by Yorick? We will approve the PR right after. Thanks

Reply:
> @Jamiet19 can you look at providing a description as highlighted by Yorick? We will approve the PR right after. Thanks

I've updated the descriptions for all parameters that didn't already have them. I've also added another parameter I discovered was missing.